### PR TITLE
[terraform-users] introduce new integration

### PR DESCRIPTION
**Pull request by maorfr**

### CODE REVIEW

File: utils/smtp_client.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    _client.sendmail(_username, to, msg.as_string())
+
+
+def send_mails(mails):
+    global _client
+
+    init_from_config()
+    for name, subject, body in mails:
+        send_mail(name, subject, body)
+    teardown()
```
Comment (jmelis) [inline utils/smtp_client.py]: maybe try/finalize if we really need to run the teardown

File: reconcile/terraform_users.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+TF_QUERY = """
+{
+  roles: roles_v1 {
+    users {
+      redhat_username
+      public_gpg_key
+    }
+    aws_groups {
+      ...on AWSGroup_v1 {
```
Comment (jmelis) [inline reconcile/terraform_users.py]: this is not needed (the `... on ` )

File: utils/smtp_client.py (hunk 1 above)
Comment (maorfr) [reply utils/smtp_client.py]: fixed

File: reconcile/terraform_users.py (hunk 2 above)
Comment (maorfr) [reply reconcile/terraform_users.py]: fixed

**Pull request merged**